sequencer/src/block2.rs

```diff
@@ -506,53 +522,46 @@ mod test {
     };
     use async_compatibility_layer::logging::{setup_backtrace, setup_logging};
     use jf_primitives::vid::VidScheme;
+    use rand::RngCore;
 
     #[test]
     fn basic_correctness() {
         // play with this
         let test_cases = vec![
-            // 3 non-empty txs
-            vec![
-                vec![0, 1, 2, 3, 4],
-                vec![5, 6, 7, 8, 9, 10, 11, 12],
-                vec![13, 14, 15, 16, 17, 18, 19, 20],
-            ],
-            // 1 empty tx at the beginning
-            vec![
-                vec![],
-                vec![5, 6, 7, 8, 9, 10, 11, 12],
-                vec![13, 14, 15, 16, 17, 18, 19, 20],
-            ],
-            // 1 empty tx in the middle
-            vec![
-                vec![0, 1, 2, 3, 4],
-                vec![],
-                vec![13, 14, 15, 16, 17, 18, 19, 20],
-            ],
-            // 1 empty tx at the end
-            vec![vec![0, 1, 2, 3, 4], vec![5, 6, 7, 8, 9, 10, 11, 12], vec![]],
-            // 1 nonempty tx
-            vec![vec![0, 1, 2, 3, 4]],
-            // 1 empty tx
-            vec![vec![]],
-            // zero txs
-            vec![],
+            vec![5, 13, 21],        // 3 non-empty txs
+            vec![0, 8, 16],         // 1 empty tx at the beginning
+            vec![5, 5, 13],         // 1 empty tx in the middle
+            vec![5, 13, 13],        // 1 empty tx at the end
```

Comment:
```suggestion
            vec![5, 13, 0],        // 1 empty tx at the end
```

Reply:
addressed in https://github.com/EspressoSystems/espresso-sequencer/pull/846#discussion_r1425946213